docs/geedocs/docsrc/answer/fusionAdministration/confTaskRulesForFusionPerf.rst

```diff
@@ -410,6 +410,21 @@ Configure Fusion performance
       - ``ALL_GRAPH_OPS`` or ``2``: use all new operations regardless of
         performance impact
 
+      .. rubric:: Use Minification
+
+      Included in ``/gevol/assets/.config/misc.xml`` is a
+      ``UseMinification`` option that controls whether terrain and imagery 
+      levels are generated from the previous level or from the original source
+      file.  This allows reducing dependencies between levels in favor of 
```

Comment:
My understanding of this change is slightly different. Not using minification reduces dependencies between levels which reduces memory usage, but it may reduce the performance of some tasks. Basically, system manager does less work but packgen tasks do more work. However, that is based on code inspection, not testing, so your description may be closer to the truth.